yas3fs/__init__.py

```diff
@@ -2803,7 +2825,7 @@ def chown(self, path, uid, gid):
                 logger.debug("chown '%s' '%i' '%i' ENOENT" % (path, uid, gid))
                 raise FuseOSError(errno.ENOENT)
             attr = self.get_metadata(path, 'attr')
-            if attr < 0:
+            if not attr:
```

Comment:
This isn't the same -- was that intentional?

Reply:
On python 3, you can't do `if attr < 0`.  `attr` is a dict, and can't be compared to 0.  I can change it to `if len (attr) < 0` if you prefer, but if `attr` is None or an empty dict, `if not attr` will be falsey

Reply:
Ah! Ok, that makes total sense now. I thought that was an int based on the comparison. Good change.